### PR TITLE
Standardized BLE api docs

### PR DESCRIPTION
**Pull request by mbartling**
* Simplified BLE api docs and standardized
* Cleaned up wording
* Added high level usage section
* Suggest adding tutorial around the LED examples

### CODE REVIEW

File: docs/refs/api/connectivity/ble.md [hunk 1]
```diff
@@ -1,40 +1,29 @@
 #### Bluetooth Low Energy (BLE)
 
-Bluetooth low energy (BLE) is a very low power wireless technology standard for personal area networks. Typical applications of BLE are health care, fitness trackers, beacons, smart home, security, entertainment, proximity sensors, industrial and automotive.
+Bluetooth low energy (BLE) is a low power wireless technology standard for building personal area networks. Typical applications of BLE are health care, fitness trackers, beacons, smart home, security, entertainment, proximity sensors, industrial and automotive.
 
-Arm Mbed BLE, also called `BLE_API`, is the Bluetooth Low Energy software solution for Mbed. Many targets support Mbed BLE. Developers can use it to create new BLE applications.
+Arm Mbed BLE, also called `BLE_API, is the Bluetooth Low Energy software solution for Mbed. Many mbed [targets](https://developer.mbed.org/platforms/?mbed-enabled=15&connectivity=3) and [components](https://developer.mbed.org/platforms/?mbed-enabled=15&connectivity=3) support Mbed BLE. Developers can use it to create new BLE enabled applications.
```
Comment (AnotherButler) [inline docs/refs/api/connectivity/ble.md]: Please capitalize Mbed.

File: docs/refs/api/connectivity/ble.md [hunk 2]
```diff
@@ -1,40 +1,29 @@
...
-        * The [heart rate monitor](https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-ble-HeartRate/) example demonstrates how to build a heart rate sensor that can be connected and monitored by a BLE client such as your phone.
-        * The [LED](https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-ble-LED/) and [LED blinker](https://developer.mbed.org/teams/mbed-os-examples/code/mbed-os-example-ble-LEDBlinker/) are a single example, which demonstrates how a client (LED) and a server (LED blinker) work together over BLE.
+[![View code](https://www.mbed.com/embed/?type=library)](https://docs.mbed.com/docs/mbed-os-api/en/mbed-os-5.5/api/classBLE.html)
 
-    <span>**Tip:** Despite the differences between the different Mbed OS versions, there is only **one** version of Mbed BLE, and it is easy to move code from one version of the OS to another. Choose the sample you use according to the version of Mbed OS supported by your development board.</span>
+##### Usage
```
Comment (AnotherButler) [inline docs/refs/api/connectivity/ble.md]: Query: Is there a better place to put this?

File: docs/refs/api/connectivity/ble.md (hunk 1 above)
Comment (AnotherButler) [inline docs/refs/api/connectivity/ble.md]: Please include your ` mark at the end of "BLE_API" to put it in code format.

File: docs/refs/api/connectivity/ble.md (hunk 2 above)
Comment (mbartling) [reply docs/refs/api/connectivity/ble.md]: This usage section is meant to mirror the other connectivity docs. I wanted this to be standard so it's easier to follow

Comment (mbartling) [reply docs/refs/api/connectivity/ble.md]: Also the stuff in red doesn't really need to be in the docs

**Pull request merged**